app/build/resources/linux/mailspring.desktop.in

```diff
@@ -7,5 +7,5 @@ Icon=mailspring
 Type=Application
 StartupNotify=true
 StartupWMClass=<%= productName %>
-Categories=GNOME;GTK;Network;Email;
+Categories=GNOME;GTK;Network;Email;Productivity;Communication;
```

Comment:
How does even mailspring assign itself as a GNOME & Gtk application? :o

Reply:
Never said it was a comment for you 😅

Reply:
👋 hey guys! You know honestly, I think I copied this file from somewhere to create it ages and ages ago - I'm really not sure what those GNOME + GTK entries do, if anything. Thanks for fixing this @bilelmoussaoui!

Reply:
@bengotow those tags are used either on a search of the system itself or in the software center. The application wasn't made using GNOME technologies (Gtk and other libraries around it), those two categories could be removed from there